README.md

```diff
@@ -29,6 +29,7 @@ import App from './components/App';
 ReactDOM.render(
   <SipProvider
     host="sip.example.com"
+    suffix="ws" // This value is appended to host string for websockets (e.g. wss://server:7443/ws); null by default
```

Comment:
How bout host, then port and then suffix? Would be great shorten the comment too. Maybe 
`Path in socket URI (e.g. wss://sip.example.com:7443/ws); null bu default`?

And what if we rename `suffix` into `path`? Sounds more conventional.